controllers/users/updateAvatar.js

```diff
@@ -0,0 +1,34 @@
+//export const updateAvatar = () => {};
+import { User } from "../../service/schemas/user.schemas.js";
+import path from "path";
+import fs from "node:fs/promises";
+import Jimp from "jimp";
+
+const updateAvatarsDir = path.join(process.cwd(), "public/avatars");
+
+export const updateAvatar = async (req, res, next) => {
+  const { _id: user } = req.user;
```

Comment:
Zostałbym przy user ID
```suggestion
  const { _id: userId } = req.user;
```

Reply:
Bo inaczej można się pomylić i myśleć sobie że ty to masz użytkownika i całe dane o nim a nie tylko samo ID